# ROBOT'S POSITION FOR SIMULATION
robot_pose_x: 0.90
robot_pose_y: 0.62
robot_pose_w: 0.0
